papers/yellow/main.tex

```diff
@@ -587,6 +562,10 @@ \subsection{\Coin{} Format}
     \item{\em bytes of a script. } To see how the script is serialized, see (\knote{link to \sigm language expressions
     serialization}). The script is to be represented as register R1 by wrapping its bytes are constant array of constant
     bytes.
+    \item{\em number of assets } which box is carrying. Represented as a one-byte signed integer.
+    \item{\em assets}. A box can carry multiple assets. A single record in this field is represented as a tuple

```

Comment:
Up to 4 assets?

Reply:
@catena2w why this detail is important in serialization description ?